src/excalidraw-app/index.tsx

```diff
@@ -273,7 +273,10 @@ const PlusLinkJSX = (
 
 const ExcalidrawWrapper = () => {
   const [errorMessage, setErrorMessage] = useState("");
-  const currentLangCode = languageDetector.detect() || defaultLang.code;
+  let currentLangCode = languageDetector.detect() || defaultLang.code;
```

Comment:
The new `languageDetector.detect()` might return a list of strings.. in that case, get the first one.